src/sentry/apidocs/api_ownership_stats_dont_modify.json

```diff
@@ -18,6 +18,8 @@
     "issues": {
```

Comment:
I have a fix for this file generation. If you're reviewing this PR you can ignore changes in this file.